### PR TITLE
Add issue and PR templates

### PR DESCRIPTION
**Pull request by JDW818**
Partially resolves checklist in Issue #84.

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Steps to reproduce:
+<!---
+List the steps that took you on your journey to discovering this bug! Include
+hyperlinks so we can go on the same journey. e.g.:
+* Go to [this Looker/Mode/Periscope report](www.link_to_report.com).
+* Also go to [this GA dashboard](www.link_to_ga.com).
+-->
+
+## Expected resuts:
```
Comment (jtcohen6) [inline .github/issue_template.md:9]:
```suggestion
## Expected results:
```

**Review by jtcohen6 — COMMENTED** [review]
Thanks for the PR to add these, @JDW818!

There are a few things in the issue template that feel more relevant to an organization's private dbt project than a public package used by many different organizations.

I like what Claire recently did to create a bug report and PR Template in the Segment package (open PR): https://github.com/fishtown-analytics/segment/pull/45

We could also add a feature request template, similar to the one we've got [here](https://github.com/fishtown-analytics/dbt/blob/dev/octavius-catto/.github/ISSUE_TEMPLATE/feature_request.md) for dbt.

**Comment by jtcohen6** [comment]
I just remembered that I included a bug report + PR template (exact copies of the ones in that Segment package PR) in an omnibus PR (#79) that brings the Snowplow package up to speed in a number of respects. That PR is still waiting on review from @sanjanasen96, or another brave volunteer :)

To that end, I just updated #84 to note that merging #79 should check off all remaining boxes.

**Comment by JDW818** [comment]
@jtcohen6 awesome! Closing this.

**Pull request closed without merging**